guides/source/active_record_migrations.md

```diff
@@ -674,10 +674,9 @@ Column modifiers can be applied when creating or changing a column:
 
 * `comment`      Adds a comment for the column.
 * `collation`    Specifies the collation for a `string` or `text` column.
-* `default`      Allows to set a default value on the column. Note that if you
-  are using a dynamic value (such as a date), the default will only be
-  calculated the first time (i.e. on the date the migration is applied). Use
-  `nil` for `NULL`.
+* `default`      Declares a default value for the column when a value is not
+  provided. Assigning any value (including `nil`) will override a column's
+  default value. Use `nil` for `NULL`.
```

Comment:
It's not that you're overriding the value, but the database simply accepts the value you provided (in this case `nil`) and therefore doesn't need to use the default.

Reply:
Using "override" here isn't appropriate. I'm hoping to emphasize that assigning a `nil` value will not incorporate or involve the default value at all. 

Would it be more clear to omit the `any value` part, and instead emphasize that assigning `nil` will not "fall back" (is "use" more clear language?) to the default value?